### PR TITLE
1126 dropdown options

### PR DESCRIPTION
**Pull request by dexguitar**

### CODE REVIEW

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/DropdownOptionsTests.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            validateAlert(is(linkArray[i - 1] + APPENDIX));
+            dropdown.collapse();
+        }
+    }
+
+    @Test
+    public void dropdownContentTest() {
+        offsetDropdown.expand();
+        offsetDropdown.assertThat()
+                .itemValues(is(Arrays.asList(OFFSET_DROPDOWN_CONTENTS)))
```
Comment (AUlasen) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/DropdownOptionsTests.java]: You can use:
`.itemValues(OFFSET_DROPDOWN_CONTENTS)`

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/DropdownOptionsTests.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+                .hasItems(ITEM_ONE)
+                .hasItems(ITEM_TWO)
+                .hasItems(ITEM_THREE);
+
+        referenceDropdown.expand();
+        referenceDropdown.assertThat()
+                .itemValues(is(Arrays.asList(REFERENCE_DROPDOWN_CONTENTS)))
```
Comment (AUlasen) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/DropdownOptionsTests.java]: You can use:
`.itemValues(REFERENCE_DROPDOWN_CONTENTS)`

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/DropdownOptionsTests.java [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+                .hasItems(ITEM_ONE)
+                .hasItems(ITEM_TWO)
+                .hasItems(ITEM_THREE)
+                .hasItems(ITEM_FOUR);
+    }
+
+    @Test
+    public void optionsCssTest() {
+        offsetDropdown.expand();
+        assertThat(offsetDropdown.core().childs().get(1).getAttribute(DATA_OFFSET), is(DATA_OFFSET_VALUE));
```
Comment (AUlasen) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/DropdownOptionsTests.java]: You should use JDI asserts:
`.assertThat().attr()`

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/DropdownOptionsTests.java [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+        offsetDropdown.collapse();
+
+        referenceDropdown.expand();
+        assertThat(referenceDropdown.core().childs().get(2).getAttribute(DATA_REFERENCE), is(DATA_REFERENCE_VALUE));
```
Comment (AUlasen) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/DropdownOptionsTests.java]: You should use JDI asserts:
`.assertThat().attr()`

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/DropdownOptionsTests.java (hunk 1 above)
Comment (dexguitar) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/DropdownOptionsTests.java]: Resolved

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/DropdownOptionsTests.java (hunk 2 above)
Comment (dexguitar) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/DropdownOptionsTests.java]: Resolved

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/DropdownOptionsTests.java (hunk 3 above)
Comment (dexguitar) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/DropdownOptionsTests.java]: Resolved

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/DropdownOptionsTests.java (hunk 4 above)
Comment (dexguitar) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/DropdownOptionsTests.java]: Resolved

**Pull request merged**